Sources/FocusEntity/FocusEntity.swift

```diff
@@ -155,6 +154,9 @@ open class FocusEntity: Entity, HasAnchoring, HasFocusEntity {
   /// Indicates if the square is currently changing its alignment.
   internal var isChangingAlignment = false
```

Comment:
One last thing, let's set this to `public internal(set)`, as it's actually used now so important if someone is using it to place objects at the focus entity position (with orientation)

Reply:
Good point, making the change.